buddyboss/buddyboss-platform
# Fix Video does not play on specfic server

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2700 

### How to test the changes in this Pull Request:

Video does not play on specfic server after added extension
Siteground- Videos Not working
Kinsta - Videos Not working

1. Upload video to Activity feed
2. Try to play video
3. symblink should be enabled
4. See error

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
